bin/loaddump

```diff
@@ -38,7 +38,27 @@ if [ ! -f $1 ]; then
     exit
 fi
 
-eval $(parse_yaml app/config/parameters.yml "")
+# Manually parse DATABASE_URL from .env.local file
+
+database_url=$(grep "^DATABASE_URL" ./.env.local)
```

Comment:
@fightingtexasaggie the .env file now has parameter called: `DATABASE_NAME`

Reply:
I think it is fine, we anyways need other parameters from the url